packages/dashboard/src/app/router/route.js

```diff
@@ -63,7 +63,7 @@ export function resolveRoute(route) {
      * If the url has no root append it to the current route and create
      * a nested root within the context of the dashboard app.
      */
-    return `${window.location.hash}/${route}`;
+    return `${globalThis.location?.hash}/${route}`;
```

Comment:
On the server `globalThis.location?.hash` is `undefined`, so this will return `undefined/${route}`

Could mitigate this using something like

```suggestion
    const hash = globalThis.location?.hash || '';
    return `${hash}/${route}`;
```